doc/man1/openssl.pod

```diff
@@ -755,6 +755,26 @@ This file can be used in a subsequent command invocation.
 
 =back
 
+=head2 Provider Options
+
+All cryptographic operations in OpenSSL 3.0 are provider based.  These options
```

Comment:
This is misleading. Even if we ignore legacy codepaths in EVP that will be removed before release, there are the engines and the legacy APIs.